### PR TITLE
arangodb 3.7.3

### PR DESCRIPTION
**Pull request by KVS85**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by fxcoudert** [comment]
Putting on hold while we merge #62560

**Comment by fxcoudert** [comment]
Please report the issue upstream, and use a patch from their repo if possible.

**Comment by KVS85** [comment]
> Please report the issue upstream, and use a patch from their repo if possible.

Thanks. It's our own repo but we can't switch to another codebase (3.7.3 is already tagged and built). We didn't notice that on our side until `clang` at 10.15 was updated. That's already fixed in main branch.

File: Formula/arangodb.rb [hunk 1]
```diff
@@ -27,6 +27,8 @@ class Arangodb < Formula
       revision: "e32307e9ae5a0046214cb066355a8577e6fc4148"
   end
 
+  patch :DATA
```
Comment (fxcoudert) [inline Formula/arangodb.rb:30]: We can pick up the commit directly from the tree like this:

```suggestion
  patch do
    url "https://github.com/arangodb/arangodb/commit/e66bf335.patch?full_index=1"
    sha256 "whatever-value"
  end
```

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/arangodb.rb (hunk 1 above)
Comment (KVS85) [reply Formula/arangodb.rb]: Thanks for the hint. Done.

**Comment by SMillerDev** [comment]
Patch sha checksum doesn't match what you added, please check.

**Comment by fxcoudert** [comment]
- Needs to be rebased on top of master (to remove the `revision 1` introduced by the python update).
- Please add a comment on top of the patch `# Fix compilation with Xcode 12, remove in next release`

**Comment by KVS85** [comment]
> - Needs to be rebased on top of master (to remove the revision 1 introduced by the python update).
>  - Please add a comment on top of the patch # Fix compilation with Xcode 12, remove in next release

Done.

**Review by fxcoudert — APPROVED** [review]

**Pull request closed without merging**